apps/examples/webserver/webserver_main.c

```diff
@@ -114,9 +135,15 @@ void http_get_root(struct http_client_t *client, struct http_req_message *req)
 	http_keyvalue_list_add(&response_headers, g_httpconnect, g_httpcnlost);
 
 	printf("===== GET_ROOT CALLBACK url : %s =====\n", req->url);
-	if (http_send_response(client, 200, msg, &response_headers) < 0) {
-		printf("Error: Fail to send response\n");
+
+	if (resp_type == 1) {
```

Comment:
where is resp_type changing here? or it it for temporary change

Reply:
Temporary Change as per requirement. I will add the comment